docs/source/technical_tutorials/user_data/stac_metadata.rst

```diff
@@ -0,0 +1,174 @@
+         role to the “MAAP” provider.
+      -  Example: if data are hosted in the NASA CMR, the “host” role
+         would be added to the respective NASA DAAC.
+
+-  ``keywords``: **Required**. Keyword(s) that describe the parameters
+   provided in the data.
+
+   -  This is a GCMD-controlled field: `Science Keywords CSV
+      file <https://gcmd.earthdata.nasa.gov/kms/concepts/concept_scheme/sciencekeywords/?format=csv>`__.
+
+-  ``links``: **Required**. Resources related to a collection. This
+   should include a link to access the data. Other links to consider
+   adding: user’s guide, a dataset landing page, ATBD, read-me, the
+   project home page, relevant online data tools, etc.
+-  ``extent``:
+
+   -  ``spatial``: **Required**. The spatial extent of the collection,
+      which should encompass all items.
+   -  ``temporal``: **Required**. The temporal extent of the collection,
+      which should encompass all items.
+
+-  ``summaries``:
+
+   -  ``platforms``: **Required**. Platforms used for data aquisition.
+
+      -  This is a GCMD-controlled field: `Platforms CSV
+         file <https://gcmd.earthdata.nasa.gov/kms/concepts/concept_scheme/platforms?format=csv>`__
+
+   -  ``instruments``: **Highly recommended**. Instruments used for data
+      aquisition.
+
+      -  This is a GCMD-controlled field: `Instruments CSV
+         file <https://gcmd.earthdata.nasa.gov/kms/concepts/concept_scheme/instruments/?format=csv>`__
+
+   -  ``mission``: **Highly recommended**. The scientific endeavor(s)
+      with which the collection is associated, including but not limited
+      to field or flight campaigns, projects, missions, scientific
+      programs, etc.
+   -  ``gsd``: **Highly recommended**. The spatial resolution of the
+      data in meters.
+   -  ``processing:level`` **Required**. The level at which the data in
+      the collection are processed, with values ranging from L0 to L4.
+
+      -  MAAP recommends using NASA’s Earth Observing System Data and
+         Information System (EOSDIS) processing levels. For more
+         information, please see `Data Processing
+         Levels <https://www.earthdata.nasa.gov/learn/earth-observation-data-basics/data-processing-levels>`__.
+
+-  ``license``: **Required**. License of the collection and its
+   associated items. The license should be permissive or open.
+
+   -  For more information and resources about licenses, see `Open
+      Sourcing
+      Code <../../science/oss_documentation/doi_and_licensing.ipynb>`__.
+
+-  ``sci:citation``: **Highly recommended**. The collection citation.
+-  ``item_assets``:
+
+   -  ``type``: **Required**. The media type of the asset.
+
+      -  `Common Media Types in
+         STAC <https://github.com/radiantearth/stac-spec/blob/master/best-practices.md#common-media-types-in-stac>`__
+
+-  ``stac_version``: **Required**. The STAC version the collection
+   implements.
+
+STAC Item-Level Metadata Fields
```

Comment:
Link to https://github.com/radiantearth/stac-spec/blob/master/item-spec/item-spec.md